frontend/src/sass/styles.scss

```diff
@@ -2,6 +2,50 @@
 
 @import '~bootstrap/scss/bootstrap';
 
+@import './animations';
+@import './mixins';
+@import './variables';
+
+
+/* Global styles */
+
 html {
-  font-size: 16px;
+  font-size: $base-font-size;
+}
+
+.loader- {
```

Comment:
I don't know also if we want to store these styles in `styles.scss`, because that is very specific element, that we will style using it. What do You think about moving it to separate file (e.g. `loder.scss`)?

Reply:
That's a good idea. I will actually try to make it more extendable, so we can reuse the loader with different sizes accross the App.